web/docs/develop-vast/contribution-workflow/notebooks.md

```diff
@@ -0,0 +1,56 @@
+# Notebooks
+
+We use [Jupyter notebooks](https://jupyter.org/) as vehicle for self-contained
+guides that illustrate how to use VAST. These notebooks (will) run in CI and
+produce outputs as side effect, which we capture and include in the
+documentation under the section
+[Try VAST](/docs/try-vast).
```

Comment:
should we make it clearer that we want the notebooks to be written in `md` format, or is it actually not a constraint we want to set?

Reply:
Good point, I added a section that covers this.